source/_includes/custom/footer.html

```diff
@@ -2,11 +2,11 @@
   <div class="container-fluid">
     <div class="row">
       <div class="span2">
-          <img src="/images/cerner_small_logo_white.png" alt="Cerner Logo" />
+          <img src="/images/Cerner_White_Horizontal.png" alt="Cerner Logo" />
```

Comment:
Nit: I would have kept the same naming pattern of the image, but versioned it (ex. cerner_small_logo_white_v2.png). However, there isn't a consistent naming pattern for static content.

Reply:
Ah, yea I just downloaded the file from our assets team and used it as is, I didn't even think about renaming it.